src/main/java/com/github/peggybrown/speechrank/InMemoryConferencesRepository.java

```diff
@@ -19,21 +17,17 @@
 
     private Importer importer;
     private List<Conference> conferences;
-    private List<Year> years;
+    private static final List<Year> YEARS = initYears();
```

Comment:
why did you make it static? is there any specific reason?

Reply:
This field belongs rather to class not particular object. Shouldn't it have only one instance through?

Reply:
I'm not arguing that it isn't as you say (I'm not 100% sure, though), but in the end the `InMemoryConferencesRepository` class would only have one instance in the application. Taking that into consideration, and also the fact that other fields are non static, I'd personally leave it as it was. In my opinion itit can confuse someone right now - as it confused me at first.